spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/support/converter/JacksonMessageConverter.java

```diff
@@ -0,0 +1,61 @@
+/*
+ *  Copyright 2018 original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.springframework.cloud.gcp.pubsub.support.converter;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.pubsub.v1.PubsubMessage;
+
+import org.springframework.integration.support.json.JacksonJsonUtils;
+
+/**
+ * A message converter using Jackson JSON.
+ *
+ * @author Chengyuan Zhao
+ */
+public class JacksonMessageConverter implements PubSubMessageConverter {
+
+	private final ObjectMapper objectMapper;
+
+	/**
+	 * Constructor
+	 * @param trustedPackages the packages trusted for deserialization in addition to the
+	 * default trusted packages listed in {@link JacksonJsonUtils}.
+	 */
+	public JacksonMessageConverter(String... trustedPackages) {
```

Comment:
There must be a separate ctor which allows end-user to inject his/her own `ObjectMapper`.
Seems for me we had such a discussion in other place in this PR...

Reply:
done

Reply:
Please, don't send us this kind of spam emails with `done` word.

The proper commit message is enough for me to understand that you agree for the changes request.

That makes sense to comment when you have some other opinion.

I have thousands emails from hundred repos on GH I follow per day, so extra noise in my mail box decrease my efficiency :smile:

Reply:
My bad. I won't do that anymore. We do that internally to let you know that your comments addressed.